src/tutorials/composer-auth.md

```diff
@@ -46,10 +46,17 @@ Then in your build hook set the authentication information for the remote server
 hooks:
     build: |
         set -e
-        composer config my-private-repos.example.com $COMPOSER_KEY $COMPOSER_PASSWORD
+        composer config http-basic.my-private-repos.example.com $COMPOSER_KEY $COMPOSER_PASSWORD
         composer install --no-dev --prefer-dist --no-progress --no-interaction --optimize-autoloader
   ```
 
-The specific switches in the `composer install` line are what Platform.sh would run by default, but you can customize that as needed.  They important part is the `composer config` line, which tells composer to use the key and password from the environment when connecting to `my-private-repos.example.com`.
+The specific switches in the `composer install` line are what Platform.sh would run by default, but you can customize that as needed.  They important part is the `composer config` line, which tells composer to use the key and password from the environment when connecting to `my-private-repos.example.com`, and to do so using `http-auth`.  (See the [Composer documentation](https://getcomposer.org/doc/06-config.md#http-basic) for other possible authentication options.)
 
 From here on, everything should proceed as normal.  Composer will download all listed packages from whatever repositories are appropriate, authenticating against them as needed, build the application, and then it will deploy as normal.  Because the variables are defined above to not be visible at runtime they will not appear at all in the running application.
+
+
+## Private repository hosting
+
+Typically, a private dependency will be hosted in a private Git repository.  While Platform.sh supports [synchronizing the primary repository](/development/private-repository.md) with a private third party, that does not apply for third party dependencies that may have their own authenticated repository.  For that reason, pulling a dependency directly from a third party Git repository that requires authentication is not supported.
```

Comment:
> pulling a dependency directly from a third party Git repository that requires authentication is not supported

Huh? It is, if you can persuade the third party to add your project's deploy key.

Maybe it's the definition of "third party" that's not clear?